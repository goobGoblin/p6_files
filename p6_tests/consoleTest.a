simpleConsoleTest : () -> void {
  x : int;
  
  toconsole "Message 1";
  fromconsole x;
  toconsole x;
}
