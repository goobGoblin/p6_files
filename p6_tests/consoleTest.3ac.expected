[BEGIN GLOBALS]
simpleConsoleTest
str_0 "Message 1"
[END GLOBALS]
[BEGIN <init> LOCALS]
[END <init> LOCALS]
fun_<init>: enter <init>
lbl_0:      leave <init>
[BEGIN simpleConsoleTest LOCALS]
x (local var of 8 bytes)
[END simpleConsoleTest LOCALS]
fun_simpleConsoleTest: enter simpleConsoleTest
            WRITE str_0
            READ [x]
            WRITE [x]
lbl_1:      leave simpleConsoleTest
